examples/http-certification/custom-assets/README.md

```diff
@@ -197,9 +208,6 @@ fn certify_asset_response(
                 asset_tree_path,
                 &certification,
             ));
-
-            // set the canister's certified data
-            set_certified_data(&http_tree.root_hash());
```

Comment:
This change is unrelated to the main change, I just noticed while I was here that we were updating the certified variable every time an asset was certified, but they're all certified on init and upgrade so we can just update this once after all files are certified.